src/Compiler/FSComp.txt

```diff
@@ -1643,4 +1643,5 @@ reprStateMachineInvalidForm,"The state machine has an unexpected form"
 3520,invalidXmlDocPosition,"XML comment is not placed on a valid language element."
 3521,tcInvalidMemberDeclNameMissingOrHasParen,"Invalid member declaration. The name of the member is missing or has parentheses."
 3522,tcAnonRecdDuplicateFieldId,"The field '%s' appears multiple times in this record expression."
-3523,tcAnonRecdTypeDuplicateFieldId,"The field '%s' appears multiple times in this anonymous record type."
\ No newline at end of file
+3523,tcAnonRecdTypeDuplicateFieldId,"The field '%s' appears multiple times in this anonymous record type."
+3524,noArgumentsAreAllowedInDelegatePrimaryConstructor,"No argument are allowed in delegate primary constructor."
```

Comment:
Perhaps we could reuse FS0552 here instead of adding another error number?

Reply:
Humm. I could not find the  FS0552 on the list. if you can point me where is I can definitely reuse that :)

Reply:
It's defined here:
https://github.com/dotnet/fsharp/blob/b5c1b447b3a6fadbae8e9d2f3012f0170bbec780/src/Compiler/FSComp.txt#L414